docs/reference/config.md

```diff
@@ -145,13 +145,13 @@ commitAuthor:
 commitMessage: "chore(release): this is a custom release message"
 ```
 
-## DirtyFiles
+##IncludeArtifacts
```

Comment:
This does not follow the pattern of this file, IncludeArtifacts is not a top level config option.  We already document this within the `## git` section, so I don't think we need this section.